spec/eslint.config.mjs

```diff
@@ -0,0 +1,9 @@
+import globals from "globals";
+
+export default [{
+    languageOptions: {
+        globals: {
+            ...globals.jest,
+        },
+    },
+}];
```

Comment:
The eslint 9 flat config should be just that, a flat config. Let's merge this into the top level config file.